File: Java/Iniciante/1002 - Área do Círculo..java
/*
 * Autor: Pierre Vieira
 * Data de submissão: 19/01/2020 16:50:21
 */
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        double r;
        r = teclado.nextDouble();
        System.out.printf("A=%.4f\n", 3.14159*r*r);
    }
}

File: C#/Iniciante/1002 - Área do Círculo.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 13/05/2020 11:07:23
*/
using System;
using System.Globalization;

namespace URI
{
    class URI
    {
        static void Main(string[] args)
        {
            double raio;
            double area;
            raio = double.Parse(Console.ReadLine());
            area = raio * raio * 3.14159;
            Console.WriteLine("A=" + area.ToString("F4", CultureInfo.InvariantCulture));
        }
    }
}
